en/px-emails.json

```diff
@@ -13,6 +13,11 @@
                 "cta": "Activate account",
                 "closing": "Enjoy!<br>IntelliZoom Panel Team"
             },
+            "excludedParticipation":{
```

Comment:
@ralayo-uz Turns out this file already had an entry for `excludedParticipation` done by the "POEditor" user. No idea if I should be doing this manually or if I should override the already existing entry. 

Only happening for `en`, the rest did not have it

Reply:
Translations aren't done by us, they are done by poeditor, so this PR is not needed really. It's already in `master` the translations for this.

Reply:
But only the ´en´ one is in `master`, right?

Reply:
where did you get all the langage translations?

Reply:
From here: https://uzjira.atlassian.net/wiki/spaces/UZProduct/pages/2997682456/IZ+Panelists+Email+-+Excluded+and+Auto-excluded+Participations, my german is a bit rusty to it by myself :D